tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "outDir": "./build",     
```

Comment:
Might be good to have `noImplicitAny: true` here

Reply:
I honestly avoided to add types where they didn't exists because I was scared not to break the app. Enabling this triggers a lot of errors. Shall I proceed with fixing them? I might need a second pair of eyes at some point if I do it

Reply:
If there are a few things to address I'd suggest opening another PR later once we've confirmed everything's ok 👍